docs/API.AMQP.md

```diff
@@ -1245,9 +1247,84 @@ These statistics include:
   status: 200,                      // Assuming everything went well
   error: null,                      // Assuming everything went well
   result: {
-    _source: {                      // Your original query
-      ...
+    _source: {}                     // Your original query
+    action: 'getLastStat',
+    controller: 'admin',
+    statistics: {
+      "YYYY-MM-DDTHH:mm:ss.mmmZ": {
+        completedRequests: {
+          websocket: 148,
+          rest: 24,
+          mq: 78
+        },
+        failedRequests: {
+          websocket: 3
+        },
+        ongoingRequests: {
+          mq: 8,
+          rest: 2
+        }
+        connections: {
+          websocket: 13
+        }
+      }
     },
+    /*
+    The requestId field you provided.
+    */
+    requestId: '<unique request identifier>'
+  }
+}
+```
+
+---
+
+### Getting the statistics from a date
+
+Kuzzle monitors its internal activities and make snapshots of them. This command allows getting the last stored statistics frame from a date.
```

Comment:
should be "This command allows getting statistics frames saved after a provided date (timestamp?)"